### DISCUSSION TITLE
v0.10.2

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#591] [Announcements]
<details>
<summary>中文版</summary>

1. 新图标 #573 （感谢 @WluhWluh）
2. 支持日语（感谢 @soulset001），土耳其语（感谢 @SerdarSaglam）和西班牙语（感谢 @AndrusGerman），修正繁体中文翻译（感谢 @a0193143）
3. 修复了 Alt+Space 和 Alt+F4 不起作用的问题
4. 启动应用时支持自定义启动参数 #575
5. Win11 中启用 Segoe UI Variable #590

</details>

1. New logo #573 (thanks @WluhWluh)
2. Support Japanese (thanks @soulset001), Turkish (thanks @SerdarSaglam) and Spanish (thanks @AndrusGerman), improve traditional Chinese translation (thanks @a0193143)
3. Fixed the issue of Alt+Space and Alt+F4 not working
4. Support custom launch parameters when starting the programs #575
5. Enable Segoe UI Variable in Win11 #590

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.10.2'>v0.10.2</a>.</em>

**Comment by AZ-6969** [comment]
portable?

**Reply by Blinue** [reply]
Just turn on the "Portable mode" option.
